MAUI/Calendar/migration.md

```diff
@@ -269,7 +269,7 @@ this.Content = calendar;
 <tr>
 <td>{{'[AgendaViewHeight](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfCalendar.XForms.SfCalendar.html#Syncfusion_SfCalendar_XForms_SfCalendar_AgendaViewHeight)'| markdownify }}</td>
```

Comment:
<img width="743" alt="Screenshot 2024-04-26 at 10 41 43 AM" src="https://github.com/syncfusion-content/maui-docs/assets/46158936/cacba8ab-4927-4dbc-8a9f-3f16f5e8a2df">

These are not covered?

Reply:
No Inline related support. So no need to update. Then YearCell and MonthCell navigating to year and month view properties.